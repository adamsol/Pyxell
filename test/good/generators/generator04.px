
k = 0

func* range(n: Int): Int def
    for x in 0...n do
        yield x
        k += 1

print [...range(0)]
print [...range(5) by 4]
print [...range(10^^7) by 2000].length

print k
