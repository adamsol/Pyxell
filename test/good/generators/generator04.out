[]
[0, 4]
5000
10000005
